plugins/linear-genome-view/src/LinearGenomeView/components/OverviewScaleBar.tsx

```diff
@@ -150,83 +200,120 @@ type LGV = Instance<LinearGenomeViewStateModel>
+              seq.reversed
+                ? classes.scaleBarContigReverse
+                : classes.scaleBarContigForward,
+            )}
             style={{
               minWidth: regionLength / scale,
               marginRight:
-                idx === displayedParentRegions.length - 1
+                idx === displayedRegions.length - 1
                   ? undefined
                   : wholeSeqSpacer,
               borderColor: refNameColor,
             }}
-            className={classes.scaleBarContig}
-            variant="outlined"
           >
+            {incompleteRegion && (
+              <div
+                className={classes.scaleBarRegionIncompleteLeft}
+                style={{
+                  backgroundImage: `linear-gradient(-225deg, ${refNameColor} 3px, transparent 1px),
+                linear-gradient(45deg, ${refNameColor} 3px, transparent 1px)`,
+                  backgroundSize: '10px 8px',
+                  backgroundRepeat: 'repeat-y',
+                }}
+              />
+            )}
             {/* name of sequence */}
             <Typography
               style={{ color: refNameColor }}
               className={classes.scaleBarRefName}
             >
               {seq.refName}
             </Typography>
-            {/* where the boxes actually get drawn   */}
+
+            {/* where the rubberband selection boxes actually get drawn */}
             {visibleRegions.map((r, visibleRegionIdx) => {
               if (
                 seq.assemblyName === r.assemblyName &&
-                seq.refName === r.refName
+                seq.refName === r.refName &&
+                r.regionNumber === idx
               ) {
+                const leftStyle = r.reversed
+                  ? (seq.end - r.end) / scale - 1
+                  : (r.start - seq.start) / scale - 1
                 return (
                   <div
                     key={`${r.key}-${visibleRegionIdx}`}
                     className={classes.scaleBarVisibleRegion}
                     style={{
                       width: Math.max((r.end - r.start) / scale, 1),
-                      left: r.start / scale - 1,
+                      left: leftStyle,
                       pointerEvents: 'none',
                     }}
                   />
                 )
               }
               return null
             })}
-            {/* the numbers */}
+            {/* the number labels */}
             {labels.map((label, labelIdx) => (
               <div
-                key={label}
+                key={`${JSON.stringify(seq)}-${label}-${labelIdx}`}
                 className={classes.scaleBarLabel}
                 style={{
                   left: ((labelIdx + 1) * gridPitch.majorPitch) / scale,
                   pointerEvents: 'none',
                   color: refNameColor,
                 }}
               >
-                {label.toLocaleString()}
+                {label.toLocaleString('en-US')}
```

Comment:
"strings" generally do not need toLocaleString, just numbers

Reply:
I must have done a replace all.. will change that.

Reply:
Wait the label is a number. These labels are the numbers that are the reference guides in the overview scale bar.

Reply:
Oh I see, maybe change the name from label to tickLabel ? Might help describe it a little

Reply:
Okay sounds good, I can do that.